.github/composite_actions/log_metric/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: 'Log Metric'
+description: 'Log data point to a metric with the provided value. If the metric is not there, it will create one.'
+inputs:
+  aws-region:
+    required: true
+    description: The AWS region
+  role-to-assume:
+    required: true
+    description: The role to assume in the STS session
+  metric-name:
+    description: 'target metric name'
+    required: true
+  value:
+    description: 'the value to log to the metric'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Configure AWS credentials
```

Comment:
What if we want to use this in a job that already has creds?

Reply:
thought about this more, I guess not a blocker here as we could add a bool param to this action to skip getting creds and could set that for some other consumer that already has creds if/when needed in future

Reply:
Yeah, thought about this more too - in what case would a job already have credentials?

Since jobs run in their own virtual image wouldn't they not be sharing credentials anyway?  

Our current actions that require credentials are pull backend and put metric, seems to make sense to keep them self contained and not having the parent jobs having credentials which could have additional security implications.

Reply:
If we wanted to use this in the integ test jobs after pulling the backends, it would already have creds in that case.